codes/typescript/test_all.ts

```diff
@@ -0,0 +1,57 @@
+import { expandGlob, type WalkEntry } from 'jsr:@std/fs';
```

Comment:
Can this script report compile bugs?

(It seems did not work before.)

Reply:
<img width="610" alt="image" src="https://github.com/krahets/hello-algo/assets/26993056/5621a64c-866e-4cb3-8ffd-f7bcc076a7de">

Emm, I meant that this script just ran the code without checking it. i.e., `test_all.ts` produced the unexpected behavior.

Reply:
The code is checked using `tsconfig.json` and executed using the `tsc` command, so the part about executing the code may be removable.

Reply:
OK